### ISSUE TITLE
Community Involvement: GitHub Audit

### ISSUE THREAD

**Issue by katoss** [alan-turing-institute/AutisticaCitizenScience#305]
## Summary
In order to make it as easy as possible for new contributors, we want to review and tidy up the GitHub repository.

## Date Due by
This issue is part of @katoss summer project work (July - August 2020). However, of course keeping the repository tidy and up to date is an ongoing task :) 

## Details
This repository on GitHub is the main external presentation of the Autistica Citizen Science Platform project and the starting point for new contributors. This is why it should be as understandable and as easy to use as possible.

## Suggestions for tasks that can fix this issue
- [X] GitHub Content Audit (list things that aren't clear, suggestions to improve the findability of documents, tidying up --> create issues for suggestions) 
- I created a fork with a suggestion for a new structure here: https://github.com/katoss/AutisticaCitizenScience based on a card sorting session I had with Georgia (https://trello.com/b/6oHZ29GA/card-sorting-acsp-github)

- [ ]  Doing a couple of 1:1 user testing sessions once the repository is tidied up and updated

## Who can help
**Everyone** who notices something that could be improved in order to make the GitHub repository more accessible. Just comment here, create an issue or fill in [this form](https://docs.google.com/forms/d/e/1FAIpQLSe_9MW3r4nsGUDxGWN24X88hqtz6q_347DC1gbMc_Yzj2Hrcg/viewform). 
This is also (and especially!) a **good task for new contributors**. If you are a new contributor and would like to take part in the **survey** (see "tasks related to the issue"), comment here or write me an email (katharina.kloppenborg@gmail.com).
- Feel free to give us feedback on the suggested new structure of the repo (https://github.com/katoss/AutisticaCitizenScience)! (however, this is still a work in progress. There are lots of readmes, links and explanations to be added. But we would love to hear what you think!)

**Comment by GeorgiaHCA** [comment]
This is brilliant work Katharina! 🌟 @JamesSCTJ would you be able to take take a look at the first version? It just has a new filing system but the content of the readme's aren't there yet? - so still a WIP but would apprecoate your thoughts!

**Comment by GeorgiaHCA** [comment]
We should also link as clearly as possible to the fork of the demo repo where Fujitsu are building. The link is here: 

https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/302

**Comment by GeorgiaHCA** [comment]
Hiya @katoss - I've had a thourough explore of your redesign and it's looking really good! Did a few edits to folder strucutre, added some READMEs, and fixed some links etc. Also made sure all the files in the original repo are there. A few things to do before putting in a PR to the main repo. Collectinng them all here before I come back to this in the morning:

- [ ]  # Make sure all the files are moved over and nothing will be lost in the PR merge - can someone double-check this for me? 
- [ ] # Decide what to do with the "Want to get involved" folder - this one isn't complete. I think it might make sense to leave this out of the PR for now, and the design it on the basis of thee website built from the repo which will be viewable ocne merged - https://alan-turing-institute.github.io/AutisticaCitizenScience/ 
- [ ] Make sure all the images are collected together in the top images folder - don't know how far you got with this @katoss but may already be complete? 
- [ ] Platform design - @jiraslama - could you maybe take a look at this folder: https://github.com/katoss/AutisticaCitizenScience/tree/master/platform-designs/wireframes and put your designs so far inside it. The "wireframe versions" folder is currently empty. Could you add your design work here or link to it here? I know a lot of your work is in comments in issues - including them in this folder would also be great! 
- [ ] Newsletters - this is a note to myself to add the more recent versions to keep this up-to-date
- [ ] There's a raw file here: https://github.com/katoss/AutisticaCitizenScience/blob/master/project-communications/fujitsu/Project.Onboarding.pptx - can this be made readable @fjThomasStanley ?
- [ ] Consent form - this is a note to myself - the one already used could be added, but also create a template version for future events. This should include Laura Crane's advice (existing currently in a PR to the main repo). Would just need to make sure the slight modifications wouldn't be a problem for the ethics approval (some of the changes are important for diversity and inclusion, like making the gender selection non-binary)
- [ ] Presentations - I've added a file containing Zenodo links to presentations in project-communications. Another note to self - this needs to be updated so it's much more readable and includes all the more recent presentations (maybe a fix forwards after merge?)
- [ ] Sprint reports - @SaimaButt here's the place for the Fujitsu sprint reports - would be brilliant if they could all be added here: https://github.com/katoss/AutisticaCitizenScience/tree/master/project-management/fujitsu-management
- [ ] Make sure all the links work! And are relative! And all the pics load!
- [ ] Some extra pics for the main README would also be a nice-to-have
- [ ] We should link to all the videos for community review - but it will need more explanation.

**Comment by GeorgiaHCA** [comment]
Sorry for that enormous list - Just trying to get it all noted before bed 🌟 ! 

I think I'll need your help @katoss, @jiraslama and either @SaimaButt or @fjThomasStanley before the PR gets made. 

@KirstieJane I knonw you are ultra time-pressured, but does this look like a good plan? Proper PR for review should be on its way shortly

**Comment by GeorgiaHCA** [comment]
Hey @JamesSCTJ - I know you were interested in helping out! If you have time, it would be great if you could give your thoughts on the redesigned repo:

https://github.com/katoss/AutisticaCitizenScience

**Comment by katoss** [comment]
Wow, thank you for all your work @GeorgiaHCA ! 
I will have a closer look at the repo when I have finished the user testing plan, but some comments already:

- yes, I had already moved all images to the **images folder** at top level (but I did not make sure yet if all the images load and links work)
- one other important thing: I moved some files from top level to other folders which might be important for the **github pages** site to work. We should definitely keep an eye on that. I remember Kirstie saying that it might be possible to move files that are necessary for github pages to the .github folder though (in case we'd like to hide them on top level). 
- about the **"want to get involved" folder:** I'd suggest we rename it to something without spaces and special characters, like e.g. "contributing" and link to the files in the main readme? As Kirstie said in the meeting we had about this, we should focus on the main readme/front page of the github pages site as entry point for new contributors?
- more on the **main readme**: we had talked about adding explanations of the two stages (building the platform - using the platform). Maybe with a little illustration? :) And yes I agree that in general some more extra pics might be a good addition!
